gson-problem/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>problem-parent</artifactId>
+        <groupId>org.zalando</groupId>
+        <version>0.24.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>gson-problem</artifactId>
+    <name>Gson Problem</name>
+
+    <properties>
+        <gson.version>2.8.6</gson.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>${gson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.zalando</groupId>
+            <artifactId>problem</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
You should get versions and scopes from the parent, so no need to repeat yourself here. (Applies to most if not all dependencies in here)

Reply:
Agreed, should I introduce `<dependencyManagement/>` in root pom containing common dependencies across all three sub-modules?

Reply:
Ah, nevermind. I was under the impression that there already is. Just leave it like this for now. I'll clean this up later, because there is most likely some duplication already that I need to remove.